.github/workflows/integrationtests.yaml

```diff
@@ -99,8 +99,8 @@ jobs:
         ports:
           - 4567:4567
         volumes:
-          - $PWD/extensions/iam/daps/src/test/resources/config:/opt/config
-          - $PWD/extensions/iam/daps/src/test/resources/keys:/opt/keys
+          - /home/runner/work/DataSpaceConnector/DataSpaceConnector/extensions/iam/daps/src/test/resources/config:/opt/config
+          - /home/runner/work/DataSpaceConnector/DataSpaceConnector/extensions/iam/daps/src/test/resources/keys:/opt/keys
```

Comment:
maybe you could use `${{ github.workspace }}`

Reply:
I don't know when and how the binding is done, i prefer to set something that will work now, since I can't try it (on bmw fork actions are disabled...)

Reply:
if it helps, I can try on my fork

Reply:
I'll merge this now but let's fix the use of hardcoded paths in a subsequent PR